content/introduction/getting-started.md

```diff
@@ -412,7 +411,7 @@ There are two types of objects that we'll use for this:
 * an **entity** is a domain object (a `Book`) that is uniquely identified by its identity,
 * a **repository** is what we use to persist, retrieve, and delete data for an entity, in the persistence layer.
 
-Entities are totally unaware of the database.
+Entities are unaware of the database.
```

Comment:
Hm, I think this is a pretty big distinction so I like the adverb for emphasis. Maybe `entirely` instead of `totally`, what do you think?